BP/history/titles/bp_k_raetia_noricum.txt

```diff
@@ -2,10 +2,19 @@
 	100.1.1 = { government = gubernatorial_government }
```

Comment:
Why the Celtic and Sol Invictus?

Reply:
IIRC he stated that the Celtic religion province was an easter egg located at Hallstatt.
For the Sol Invictus, I found sources that mention an attempt at suppressing them near Vienna made in 474 + another source mentioned by him that shows another preaching saint.

Reply:
It is true that Pannonia was Christianised during the 5th century, although the Celtic easter egg in Halstatt, while clever, is maybe a stretch here.

Reply:
Alright, I'll look into replacing that later today.